internal/state/graph/httproute_test.go

```diff
@@ -699,6 +698,9 @@ func TestBindRouteToListeners(t *testing.T) {
 					Gateway: client.ObjectKeyFromObject(gw),
 					Attachment: &ParentRefAttachmentStatus{
 						Attached: true,
+						AcceptedHostnames: map[string][]string{
+							"": {"foo.example.com"},
```

Comment:
Why is the key empty string here?

Reply:
Oh I think that was because the listener created in the test doesn't have a name. I'll need to update the test constructor with that.